ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,256 @@
+﻿[[!meta title="Getting Involved"]]
+
+## How can I get involved in the OCF?
+
+Glad you asked! Students from all backgrounds are welcome to join staff. We
+don’t interview or have a selective process for joining—you simply have to
+express interest and show up to our weekly meetings. If you want to learn more
+and be an active participant, there are multiple ways to get started:
+
+### Stay up-to-date
+
+We use a mailing list ([ocf.io/announce][announce]) to announce weekly meetings
+and recap past meetings. If you’re joining in the middle of the semester or
+missed a meeting, you can also read the mailing list archives to see what we
+went over in the past. This is a low volume mailing list (2 emails per week)
+and you can unsubscribe at any time.
+
+[[Read more about other mailing lists available to join|doc
+staff/mailing-lists]]
+
+### In-person meetings
+
+We hold two meetings every Monday:
+
+#### Board of Directors (BoD)
+
+**Mondays 7-8pm**
+
+BoD is where we discuss and vote on administrative decisions. If you are
+someboday who wants to learn about how an organization such as the OCF gets and
+spends its money, what issues the OCF needs to deal with, and all the
+administrative work that needs to be done to keep the OCF running, please come
+to BoD meetings! Meetings are always open to all. Some examples of what we
+discuss at BoD:
```

Comment:
Most of these things happen outside of BoD, we just discuss progress at BoD (and sometimes vote on things). I kind of think we shouldn't group these examples under the "in-person meetings" category. However, I think this is definitely well-written and good for now. We can think about restructuring this part later.

Reply:
ahhh this is a draft, wasn't meant to go in this commit, see the gdocs list in #administrivia for comments on this